src/uct/sm/scopy/knem/knem_ep.c

```diff
@@ -12,9 +12,20 @@
 #include <uct/sm/base/sm_iface.h>
 #include <ucs/debug/log.h>
 
+
+const char *uct_knem_ep_tx_op_str[] = {
+    [UCT_SCOPY_TX_GET_ZCOPY] = "READ",
+    [UCT_SCOPY_TX_PUT_ZCOPY] = "WRITE"
+};
+
+
 static UCS_CLASS_INIT_FUNC(uct_knem_ep_t, const uct_ep_params_t *params)
 {
     UCS_CLASS_CALL_SUPER_INIT(uct_scopy_ep_t, params);
+
+    /* This value is used to set `knem_cmd_inline_copy::write` field */
+    UCS_STATIC_ASSERT(UCT_SCOPY_TX_PUT_ZCOPY == 1);
```

Comment:
can you elaborate plz?

Reply:
pls move this assert to where we use this assumption, e.g constructing knem copy input